deployments/common/ap-policy-definition.yaml

```diff
@@ -0,0 +1,874 @@
+                      checkQueryStringLength:
+                        type: boolean
+                      checkRequestLength:
+                        type: boolean
+                      checkUrlLength:
+                        type: boolean
+                      name:
+                        type: string
+                      postDataLength:
+                        type: integer
+                      queryStringLength:
+                        type: integer
+                      requestLength:
+                        type: integer
+                      responseCheck:
+                        type: boolean
+                      type:
+                        enum:
+                        - explicit
+                        - wildcard
+                        type: string
+                      urlLength:
+                        type: integer
+                    type: object
+                  type: array
+                fullPath:
+                  type: string
+                general:
+                  properties:
+                    allowedResponseCodes:
+                      items:
+                        format: int32
+                        maximum: 999
+                        minimum: 100
+                        type: integer
+                      type: array
+                    customXffHeaders:
+                      items:
+                        type: string
+                      type: array
+                    enforcementReadinessPeriod:
+                      type: integer
+                    maskCreditCardNumbersInRequest:
+                      type: boolean
+                    trustXff:
+                      type: boolean
+                  type: object
+                header-settings:
+                  properties:
+                    maximumHttpHeaderLength:
+                      pattern: any|\d+
+                      type: string
+                  type: object
+                headers:
+                  items:
+                    properties:
+                      base64Decoding:
+                        type: boolean
+                      checkSignatures:
+                        type: boolean
+                      htmlNormalization:
+                        type: boolean
+                      id:
+                        type: string
+                      kind:
+                        type: string
+                      lastUpdateMicros:
+                        type: integer
+                      mandatory:
+                        type: boolean
+                      maskValueInLogs:
+                        type: boolean
+                      name:
+                        type: string
+                      normalizationViolations:
+                        type: boolean
+                      percentDecoding:
+                        type: boolean
+                      selfLink:
+                        type: string
+                      type:
```

Comment:
not a string. enum of wildcard|explicit

Reply:
which field are You refering to?

Reply:
"type"
Just like for cookies and filetypes that you already have